### ISSUE TITLE
Overwrite on git push

### ISSUE THREAD

**Issue by periode** [NYUAD-IM/website-panel#6]
Whenever we do a git push, we overwrite the existing JSON data, so two solutions are:
- .gitignore the public/data folder
- establish a permanent database on another heroku instance

**Comment by koktavy** [comment]
public/data and public/archive are .gitignored but files are still overwriting. Looking into it.

**Comment by jameshosken** [comment]
Try use "public/data/" and "public/archives/" (note extra slash) ?

**Comment by koktavy** [comment]
.gitignore is successful, but Heroku will not retain files written server-side on the dyno by design, since it uses an ephemeral filesystem. The Heroku docs [recommend](https://devcenter.heroku.com/articles/s3) Amazon's S3 cloud storage, which has free options I doubt this app would exceed, but requires card details in the event of overage. Outside of other suggestions, I can move forward if the department is willing to create an account.

**Comment by periode** [comment]
yeah, i think we really need a S3 storage, even for further projects. I'll talk to @craigprotzel about it.

**Comment by periode** [comment]
In the meantime, what about trying mLab https://mlab.com/ the online version of mongoDB?
